SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -79,9 +79,11 @@ class FSpatialGDKEditorToolbarModule : public IModuleInterface, public FTickable
 	void DestroySpatialDebuggerEditor();
 	void InitialiseSpatialDebuggerEditor(UWorld* World);
 	bool IsSpatialDebuggerEditorEnabled() const;
+	bool IsMultiWorkerEditorDisabled() const;
```

Comment:
Would generally pref us to go with Enabled rather than Disabled getters for consistency

Reply:
I am using this function for the toolbar menu option to display the checkbox and as the menu option is "Multi-Worker Disable" I need this to be a disable rather than enable check.
```
	InPluginCommands->MapAction(FSpatialGDKEditorToolbarCommands::Get().ToggleMultiWorkerEditor,
								FExecuteAction::CreateRaw(this, &FSpatialGDKEditorToolbarModule::ToggleMultiworkerEditor),
								FCanExecuteAction::CreateRaw(this, &FSpatialGDKEditorToolbarModule::OnIsSpatialNetworkingEnabled),
								FIsActionChecked::CreateRaw(this, &FSpatialGDKEditorToolbarModule::IsMultiWorkerEditorDisabled));
```

Reply:
I think we can just make the toggle `Enable Multi-Worker`.

Reply:
Sure I will keep it all consistent with enable everywhere.